x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -317,6 +321,15 @@ export const getAlertActions = ({
     return module === 'endpoint' && kind === 'alert';
   };
 
+  const isFromValidRule = () => {
```

Comment:
"valid rule" doesn't relay much info here, I'd say we be more explicit:
```suggestion
  const exceptionsAreAllowed = () => {
```